.github/workflows/continuous-integration.yml

```diff
@@ -17,17 +17,15 @@ jobs:
           - macos-latest
         rust_channel:
           - stable
-          - beta
-          - nightly
         include:
           - os: ubuntu-latest
             system: x86_64-linux
             experimental: false
           - os: macos-latest
             system: x86_64-darwin
             experimental: false
-          - rust_channel: nightly
-            experimental: true
+          - rust_channel: stable
```

Comment:
@Avi-D-coder You can just delete them. The previous lines set `experimental` to `false` by default already.

Reply:
@Avi-D-coder I'm sorry, "them" may have been unclear. What I meant is that you should delete the original lines:

```
- rust_channel: nightly
  experimenta: true
```

but I think we have to keep the other `experimental: false`

Reply:
@yannham  Updated.
It should work both ways since 
```yaml
    # The `== true` makes it work wether experimental has been defined or not.
    continue-on-error: ${{ matrix.experimental == true }}
```

Reply:
Oh, you're totally right. Sorry for the noise then, either way is fine.